Makefile

```
@@ -158,9 +163,28 @@ fetch-batch-gateway: ## Fetch the full batch-gateway repo at BATCH_GATEWAY_REF (
 		git -C $(BATCH_GATEWAY_DIR) checkout -q FETCH_HEAD; \
 	fi
 
+.PHONY: fetch-llm-d-async
+fetch-llm-d-async: ## Sparse-checkout only charts/async-processor from llm-d-async at LLM_D_ASYNC_REF.
```

checkmake (0.3.2)

[warning] Target body for "fetch-llm-d-async" exceeds allowed length of 5 lines (13).

(maxbodylength)